_dashboards/management/scheduled-query-acceleration.md

```diff
@@ -0,0 +1,252 @@
+
+## Prerequisites
+
+Before configuring SQA, verify that the following requirements are met:
+
+- Ensure you're running OpenSearch version 2.17 or later.
+- Ensure you have the SQL plugin installed. The SQL plugin is part of most OpenSearch distributions. For more information, see [Installing plugins]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/).
+- Ensure you have configured an Amazon S3 and Amazon EMR Serverless (needed for access to Apache Spark). 
+
+## Configuring SQA
+
+To configure SQA, perform the following steps.
+
+### Step 1: Configure the OpenSearch cluster settings
+
+Configure the following cluster settings:
+
+-  **Enable asynchronous query execution**: Set `plugins.query.executionengine.async_query.enabled` to `true` (default value):
+    ```json
+    PUT /_cluster/settings
+    {
+      "transient": {
+        "plugins.query.executionengine.async_query.enabled": "true"
+      }
+    }
+    ```
+    {% include copy-curl.html %}
+
+    For more information, see [Settings](https://github.com/opensearch-project/sql/blob/main/docs/user/admin/settings.rst#pluginsqueryexecutionengineasync_queryenabled).
+
+- **Configure the external scheduler interval for asynchronous queries**: This setting defines how often the external scheduler checks for tasks, allowing customization of refresh frequency. There is no default value for this setting so you must explicitly configure it. Adjusting the interval based on workload can optimize resources and manage costs:
+    ```json
+    PUT /_cluster/settings
+    {
+      "transient": {
+        "plugins.query.executionengine.async_query.external_scheduler.interval": "10 minutes"
+      }
+    }
+    ```
+    {% include copy-curl.html %}
+
+    For more information, see [Settings](https://github.com/opensearch-project/sql/blob/main/docs/user/admin/settings.rst#pluginsqueryexecutionengineasync_queryexternal_schedulerinterval).
+
+### Step 2: Configure a data source
+
+Connect OpenSearch to your Amazon S3 data source using the OpenSearch Dashboards interface. For more information, see [Connecting Amazon S3 to OpenSearch]({{site.url}}{{site.baseurl}}/dashboards/management/S3-data-source/).
+
+After this step, you can directly query your S3 data (the primary data source) using [Query Workbench]({{site.url}}{{site.baseurl}}/dashboards/query-workbench/).
+
+### Step 3: Configure query acceleration
+
+Configure a skipping index, covering index, or materialized view. These secondary data sources are additional data structures that improve query performance by optimizing queries on external data sources, such as Amazon S3. For more information, see [Optimize query performance using OpenSearch indexing]({{site.url}}{{site.baseurl}}/dashboards/management/accelerate-external-data/).
+
+After this step, you can [run accelerated queries](#running-an-accelerated-query) using one of the secondary data sources. 
```

Comment:
This step 3 is the same as "run accelerated queries".  Open source user should be able to use this feature without any customization since default is enabled with threshold as 5 mins. But they are able to custermize via  `Configure the OpenSearch cluster settings`